openedx/features/course_experience/views/course_home.py

```diff
@@ -35,6 +36,7 @@ def get(self, request, course_id):
         """
         course_key = CourseKey.from_string(course_id)
         course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=True)
+        CourseTabView.add_new_relic_course_key_metrics(course_key)
```

Comment:
@andy-armstrong I know this should be removed once this becomes a CourseTabView, but I guess we'll see who merges first. ;)